docs/library-specification.md

```diff
@@ -8,7 +8,7 @@ This is the specification for the Arduino library format, to be used with Arduin
 This new library format is intended to be used in tandem with **Library Manager**, available since Arduino IDE 1.6.2.
 The Library Manager allows users to automatically download and install libraries needed in their projects, with an easy
 to use graphical interface in the
-[Arduino IDE](https://www.arduino.cc/en/guide/libraries#toc3)/[Arduino IDE 2.0](https://www.arduino.cc/en/Tutorial/getting-started-with-ide-v2/ide-v2-installing-a-library#installing-a-library)
+[Arduino IDE](https://docs.arduino.cc/software/ide-v1/tutorials/installing-libraries#using-the-library-manager)/[Arduino IDE 2.0](https://www.arduino.cc/en/Tutorial/getting-started-with-ide-v2/ide-v2-installing-a-library#installing-a-library)
```

Comment:
```suggestion
[Arduino IDE](https://docs.arduino.cc/software/ide-v1/tutorials/installing-libraries#using-the-library-manager)/[Arduino IDE 2.0](https://docs.arduino.cc/software/ide-v2/tutorials/ide-v2-installing-a-library#installing-a-library)
```
This one doesn't 404 because they actually set up a redirect, but the redirect points to the wrong page.